packages/atlas/src/components/TablePaymentsHistory/TablePaymentsHistory.tsx

```diff
@@ -58,7 +60,7 @@ export const TablePaymentsHistory: FC<TablePaymentsHistoryProps> = ({ data }) =>
       })),
     [data]
   )
-  return <Table title="History" columns={COLUMNS} data={mappedData} />
+  return <Table title="History" columns={COLUMNS} data={mappedData} minWidth="750px" />
```

Comment:
Why do we need to pass width as a prop?
1. Table should probably just take full width and the rest should be handled by  the layout
2. If point above is not true for some reason, we probably should allow passing `className` to `Table` and just provide this style via styled components